lib/deck.rb

```diff
@@ -0,0 +1,40 @@
+class Deck
+  attr_reader :cards
+
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def rank_of_card_at(index)
+    cards[index].rank
+  end
+
+  def high_ranking_cards
+    high_cards = []
+    cards.each do |card|
+      if card.rank >= 11
+        high_cards << card
+      end
+    end
+    high_cards
```

Comment:
This looks great! When you're ready for another challenge, see if you can solve this with `select` instead of `each`.